src/htm/regions/SPRegion.cpp

```diff
@@ -46,7 +45,7 @@ SPRegion::SPRegion(const ValueMap &values, Region *region)
   args_.columnCount = values.getScalarT<UInt32>("columnCount", 0);
   args_.potentialRadius = values.getScalarT<UInt32>("potentialRadius", 0);
   args_.potentialPct = values.getScalarT<Real32>("potentialPct", 0.5);
-  args_.globalInhibition = values.getScalarT<bool>("globalInhibition", true);
+  args_.globalInhibition = values.getScalarT<bool>("globalInhibition", false);
```

Comment:
please use True as default, local inh is quite slow, and global is the default in the SP class.

Reply:
>>please use True as default,

Well...the problem is that this PR does change behavior slightly.  In the past the defaults from the Spec were incorporated in `values`.  The default given here was never used.  The new behavior is that the defaults in the Spec are NOT included in `values` and therefore this default is now being used.

What I did was change these values to be the same as what was given in the Spec so that the default parameters passed on to the SP algorithm are unchanged.

Now, if you want to change the default to True in this case, then you must also change the expected results in the tests.   I agree that the default here should be the same as the defaults used by SP but that sounds like a separate PR.

Reply:
> What I did was change these values to be the same as what was given in the Spec so that the default parameters passed on to the SP algorithm are unchanged. 

ok, thanks for explanation, this is fine. also for other on the same topic :+1: 

> I agree that the default here should be the same as the defaults used by SP but that sounds like a separate PR. 

yep, let's keep that for another pr